<launch> 
    <node pkg="assignment2" type="node_B" name="node_B" output="screen"/>
</launch>
